src/store/index.js

```diff
@@ -0,0 +1,7 @@
+import { createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import logger from 'redux-logger';
+import rootReducer from '../reducers';
+
+const store = createStore(rootReducer, applyMiddleware(thunk, logger));
```

Comment:
Add `window.devToolsExtension` enhancer in order to see all Redux state updates in the Chrome Redux extension. Be sure that this enhancer will be added only in Development environment (hint: use `DefinePlugin` in Webpack config )